tests/test_async_uploader.py

```diff
@@ -54,6 +55,34 @@ def test_upload(self):
             self.assertEqual(self.async_uploader.offset,
                              self.async_uploader.get_file_size())
 
+    def test_upload_empty(self):
+        with aioresponses() as resps:
+            resps.post(
+                "http://tusd.tusdemo.net/files/", status=200,
```

Comment:
Not `self.url` (it's hardcoded to an existing upload), but `self.client.url` contains an URL to a tus upload endpoint and we can use that instead.

Since we're practically mocking all the HTTP requests in this scenario, it would probably be better to use a fake URL we know that doesn't correspond to a real service (eg. `mock-tus.localdomain`), since we want the test to fail instead of contacting a real endpoint. But I'm using the same URL for consistency's sake here.